sklearn/covariance/_shrunk_covariance.py

```diff
@@ -782,6 +782,13 @@ class OAS(EmpiricalCovariance):
     np.float64(0.0195...)
     """
 
+    # TODO(1.8): Remove this method. Necessary for passing the docstring check
+    # during the deprecation of _estimator_type.
+    def __init__(self, store_precision=True, assume_centered=False):
+        super().__init__(
+            store_precision=store_precision, assume_centered=assume_centered
+        )
```

Comment:
What's going wrong if we don't do that ?

Reply:
It is complaining about passing `*args`. Speaking with @adrinjalali, we should only warn in our utils that relies on `_estimator_tags` instead to do that at the class level.

It will be much less invasive.